### DISCUSSION TITLE
(Feature request) Launchpad Mini MK2 fader preset

### DISCUSSION THREAD

**Discussion by matt9905** [helgoboss/helgobox#1222] [Ideas]
Hi,

I was interested in using my Launchpad Mini MK2 as a MIDI fader with LED feedback, but couldn't figure it out.

I [posted](https://forums.cockos.com/showpost.php?p=2810416&postcount=4504) on the realearn thread about this, and you mentioned building a controller preset that could do all of this automatically.

Thank you!

**Comment by helgoboss** [comment]
Selected for implementation in issue #1230.
